flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentITCase.scala

```diff
@@ -76,7 +76,7 @@ class TableEnvironmentITCase(tableEnvName: String, isStreaming: Boolean) extends
     TestTableSourceSinks.createPersonCsvTemporaryTable(tEnv, "MyTable")
   }
 
-  @Test
+  @TestTemplate
```

Comment:
I found some tests use `TestTemplate` annotation with `ParameterizedTestExtension`,some tests use  `Test` annotation with `ParameterizedTestExtension`, IIUC, we should use `TestTemplate` for all tests with `ParameterizedTestExtension`